benchmarks/src/main/scala/zio/chunks/ChunkArrayBenchmarks.scala

```diff
@@ -0,0 +1,48 @@
+package zio.chunks
+import zio._
+
+class ChunkArrayBenchmarks {
+
+  import java.util.concurrent.TimeUnit
+
+  import org.openjdk.jmh.annotations._
+  @State(Scope.Thread)
+  @BenchmarkMode(Array(Mode.AverageTime))
+  @OutputTimeUnit(TimeUnit.NANOSECONDS)
+  class ChunkArrayBenchmarks {
+
+    @Param(Array("100", "1000", "10000"))
+    var size: Int = _
+
+    var chunk: Chunk[Int] = _
+
+    var array: Array[Int] = _
+
+    @Setup(Level.Trial)
+    def setup() = {
+      array = Array.fill(0)(size)
+      chunk = Chunk.fromArray(array)
+    }
+
+    @Benchmark
+    def chunkFold(): Int =
```

Comment:
Let's apply renaming suggestion given above:

- `chunkFold` to `fold`
- `chunkMap` to `map`
- etc.